src/Lib/Request.php

```diff
@@ -82,7 +82,7 @@ function ($response) {
                 return $this->parseResponse($response);
             },
             // $onRejected
-            function () {
+            function ($reason) {
```

Comment:
It is unused for now, it also shows up in the idea like that why include it if i may ask?